ptftplib/notify.py

```diff
@@ -58,43 +59,43 @@ class StreamEngine(logging.StreamHandler):
     """Simple stream log handler, similar to what you get using
     logging.basicConfig."""
 
-    def __init__(self, stream, loglevel, format):
+    def __init__(self, stream, loglevel, format_):
```

Comment:
`format` is a built-in keyword.  Overwriting it can cause all sorts of issues.  I added the underscore at the end but maybe a better name could be `fmt`?